packages/contracts-bedrock/test/libraries/GasPayingToken.t.sol

```diff
@@ -0,0 +1,55 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.15;
+
+// Target contract
+import { GasPayingToken } from "src/libraries/GasPayingToken.sol";
+import { Constants } from "src/libraries/Constants.sol";
+import { Test } from "forge-std/Test.sol";
+import { Bytes } from "src/libraries/Bytes.sol";
+
+/// @title GasPayingToken_Roundtrip_Test
+/// @notice Tests the roundtrip of setting and getting the gas paying token.
+contract GasPayingToken_Roundtrip_Test is Test {
```

Comment:
Can we also add a fuzz test where the name and symbol are strings and we use the sanitize helper function to convert to bytes32 before setting and then call get and ensure that the string returned matches the input? The inputs to the fuzz function will mock the returndata from calling `ERC20.name()` and `ERC20.symbol()`. This will cover the beginning and end of the flow, ensuring the data has integrity

Reply:
good idea!